### PR TITLE
Move liquibase to dao

### PR DESCRIPTION
**Pull request by Yura09**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -52,17 +51,6 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
-        <dependency>
-            <groupId>org.liquibase</groupId>
```
Comment (antonbakalets) [inline core/pom.xml:56]: This should stay. When `dao` will be `<scope>runtime</scope>` you will have to revert this back.

File: core/pom.xml [hunk 2]
```diff
@@ -192,11 +180,6 @@
             <artifactId>google-cloud-storage</artifactId>
             <version>${cloudstorage.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-test</artifactId>
```
Comment (antonbakalets) [inline core/pom.xml:197]: This is used by controller tests.

File: dao/src/test/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Datasource
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.datasource.driverClassName=org.h2.Driver
+server.port=0
+# Liquibase
+#spring.liquibase.url=jdbc:postgresql://localhost:5432/test
+#spring.liquibase.user=${DATASOURCE_USER}
```
Comment (antonbakalets) [inline dao/src/test/resources/application.properties:9]: Please remove commented lines

File: dao/pom.xml [hunk 4]
```diff
@@ -29,5 +33,39 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-maven-plugin</artifactId>
+            <version>${liquibase.version}</version>
```
Comment (antonbakalets) [inline dao/pom.xml:39]: Please check if it is possible to add `<scope>test</scope>` as at dao level liquibase is used only to update the test database.

File: core/src/test/resources/application.properties [hunk 5]
```diff
@@ -1,60 +1,9 @@
...
-spring.datasource.driverClassName=org.h2.Driver
-server.port=0
-# Liquibase
-#spring.liquibase.url=jdbc:postgresql://localhost:5432/test
-#spring.liquibase.user=${DATASOURCE_USER}
-#spring.liquibase.password=${DATASOURCE_PASSWORD}
-spring.liquibase.enabled=true
-spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
-# Hibernate
 spring.jpa.hibernate.ddl-auto=validate
```
Comment (antonbakalets) [inline core/src/test/resources/application.properties:1]: This 4 lines can be removed. They do not make sense without `datasource`.

**Review by antonbakalets — APPROVED** [review]
Nice work. Please try to fix those minor comments.

**Review by TarasDovganyuk — APPROVED** [review]

File: dao/src/test/resources/application.properties [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+server.port=0
+
+spring.liquibase.enabled=true
+spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
+
+spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+spring.jpa.show-sql=false
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
```
Comment (antonbakalets) [inline dao/src/test/resources/application.properties:14]: Section definition comments could stay :smile:

Comment (Yura09) [reply dao/src/test/resources/application.properties:14]: Okey😄

**Review by forestik — APPROVED** [review]

**Pull request merged**